src/ansys/geometry/core/tools/prepare_tools.py

```diff
@@ -372,10 +372,8 @@ def find_and_remove_logos(
         """
         from ansys.geometry.core.designer.body import Body
 
-        if BackendType.is_linux_service(
-            self._grpc_client.backend_type
-        ) and self._grpc_client.backend_version < (26, 1, 0):
-            # not yet available on Linux until 26.1.0
+        if BackendType.is_linux_service(self._grpc_client.backend_type):
+            # not yet available in Linux
```

Comment:
Oh so it's not available - generally-speaking - on Linux? I thought it would have been available in 26R1